### PR TITLE
feat: sentry statistics

### PR DESCRIPTION
**Pull request by mcdurdin**
Integrates reporting from sentry.keyman.com into the site. For platforms, we always show the stats, but for sites, only when there are errors (to reduce noise).

Also adds a row for 'common' PRs and issues so we don't have them wrongly showing at the top.

I have added the env var to the live host. For local testing, add `export KEYMANSTATUS_SENTRY_TOKEN=...` to your localenv.sh (you'll need to generate a token of course).

Have not added self-reporting to this site yet!

### CODE REVIEW

**Comment by mcdurdin** [comment]
Question: should the error state for crashes be orange instead of red? To make it visible and yet distinct?

File: public/src/app/sites.ts [hunk 1]
```diff
@@ -8,4 +8,17 @@ export const sites = [
   's.keyman.com',
   'status.keyman.com', // ooh self referential!
   'developer.keyman.com'
-];
\ No newline at end of file
+];
+
+// TODO: this could be consolidated above
+export const siteSentryIds = {
```
Comment (darcywong00) [inline public/src/app/sites.ts]: Isn't this more appropriate in sentry.component.ts?

**Comment by darcywong00** [comment]
> For local testing, add export KEYMANSTATUS_SENTRY_TOKEN=... to your localenv.sh (you'll need to generate a token of course)

Can this get added to README.md?

File: public/src/app/sites.ts (hunk 1 above)
Comment (mcdurdin) [reply public/src/app/sites.ts]: Yeah maybe ... or some centralised constants location given it overlaps.

**Pull request merged**